packages/next/taskfile.js

```diff
@@ -310,10 +310,24 @@ export async function ncc_chalk(task, opts) {
 // eslint-disable-next-line camelcase
 externals['browserslist'] = 'next/dist/compiled/browserslist'
 export async function ncc_browserslist(task, opts) {
+  const browserslistModule = require.resolve('browserslist')
+  const nodeFile = join(dirname(browserslistModule), 'node.js')
+
+  const content = await fs.readFile(nodeFile, 'utf8')
+  await fs.writeFile(
+    nodeFile,
+    content.replace(
+      /require\(require\.resolve\(/g,
+      `__non_webpack_require__(__non_webpack_require__.resolve(`
```

Comment:
Might be good to add some comments for why replacing this?

Reply:
Added in https://github.com/vercel/next.js/pull/33890/commits/cbdb13b4a496f927696681d929f5b0dc7e87cfe0